admin/c2cgeoportal_admin/views/ogc_servers.py

```diff
@@ -188,4 +227,21 @@ def synchronize(self) -> Dict[str, Any]:
                 "report": synchronizer.report(),
             }
 
+        self._update_cache(obj)
+
         return {}
+
+    @view_config(route_name="ogcserver_clear_cache", renderer="json")  # type: ignore
+    def ogcserver_clear_cache_view(self) -> Dict[str, Any]:
+
+        self._update_cache(DBSession.query(OGCServer).filter_by(id=self.request.params.get("id")).one())
+        came_from = self.request.params.get("came_from")
+        if came_from:
+            raise pyramid.httpexceptions.HTTPFound(location=came_from)
```

Comment:
```suggestion
            return pyramid.httpexceptions.HTTPFound(location=came_from)
```

Reply:
You're sure that's working with the renderer?

Reply:
Note that pyramid.httpexceptions.HTTPFound inherit from Response class.
And this is not an error so I would like to avoid `raise` in this case.
And errors are not correctly handled by c2cwsgiutils, mainly concerning CORS.